### PR TITLE
English translation for the "IO app" page.

### LINKED ISSUE #1212: No IO APP page in English
There's no English page for IO: Italian page is at https://developers.italia.it/it/io, but there's no https://developers.italia.it/en/io

### PR DESCRIPTION
**Pull request by DevLBD**
## Description

This commit translates the IO app page in English.

## Checklist

* [ x] I followed the indications in [CONTRIBUTING.md](https://github.com/italia/developers.italia.it/blob/master/CONTRIBUTING.md)
* [ x] The documentation related to the proposed change has been updated accordingly (also comments in code).
* [ x] Have you written new tests for your core changes, as applicable?
* [ x] Have you successfully ran tests with your changes locally?
* [ x] Ready for review! :rocket:

## Fixes

Fixes #1212

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  There is a problem with the commit message
> Fix: #1212

Translated io.md
- type must be lower-case
- type must be one of [build, chore, ci, docs, feat, fix, perf, refactor, revert, style, test]
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> Begin work on translation.

Created file to begin work on the translation.
- subject may not be empty
- type may not be empty</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 1ba9f81e3b343c680c3617b9ae8454bb085177f4
</p>

File: _platforms/en/io.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+title: IO
+subtitle: The app of the public services
+logo: /assets/images/logo-io@2x.png
+external_website: https://io.italia.it
+payoff: The app of the public services
+description: io is the app of the Italian Government that will allow to all the citizens to use the national and local public services from their own smartphone in a simple, modern and secure way.
+owners:
+    - name: PagoPA S.p.A.
+      url: https://www.pagopa.gov.it/
+lang: en
```
Comment (bfabio) [inline _platforms/en/io.md:11]:
```suggestion
lang: en
ref:
  it: /it/io
```

We need this to link to the Italian page in the language switcher
(also, we need `en: /en/io` in the Italian page, to do the opposite)

**Review by bfabio — CHANGES_REQUESTED** [review]

**Comment by bfabio** [comment]
@DevLBD thanks, and congrats for your first PR :rocket:

**Comment by DevLBD** [comment]
> @DevLBD thanks, and congrats for your first PR 🚀

Thanks for your congratulations! I committed the changes you requested. Hope it is all looking good now!

Have a good day! 😃

**Pull request merged**